packages/dds/sequence/src/intervals/sequenceInterval.ts

```diff
@@ -90,7 +92,8 @@ export class SequenceInterval implements ISerializableInterval {
 		public intervalType: IntervalType,
 		props?: PropertySet,
 		public readonly stickiness: IntervalStickiness = IntervalStickiness.END,
-		public readonly isExclusive: boolean = false,
+		public readonly startSide: Side = Side.Before,
+		public readonly endSide: Side = Side.After,
```

Comment:
similar to previous comment, I would have expected `Side.Before` here as the default